### DISCUSSION TITLE
Treatment of Thysanosoria?

### DISCUSSION THREAD

**Discussion by choess** [pteridogroup/ppg#11] [Ideas]
_Thysanosoria pteridiformis_ (Ces.) C.Chr., an Irian Jaya endemic, has not, as far as I know, ever been included in a molecular phylogeny. The type is Beccari s.n. 1872 (K000647284); the only other collections made seem to be Gibbs 6162 in 1914 (K000647283) and Sands et al. 6284 and Johns et al. 8131, made in 1994. Beccari's collection has fertile fronds, with pinnae 6 mm wide, while all the others have narrower fronds not greater than 3 mm (fide Holttum in Flora Malesiana and examination of the images). Gepp's material has been regarded as a distinct species, _T. dimorphophylla_, but this was dismissed by Holttum who felt that the variation reflected fertile fronds of different ages. Nayar (in New Phytologist 65(2): 125-266 (1966)) examined the fertile frond morphology of the Lomariopsidaceae as then accepted (including various members of Tectariaceae and Dryopteridaceae with dimorphic fronds) in some detail, including _Thysanosoria_ based on Beccari's material.

_Thysanosoria_ was recombined into _Lomariopsis_ ("tentatively merged...on morphological grounds") in Christenhusz & Chase (2014) "Trends and concepts in fern classification" but was maintained as distinct in PPG I.

Is there any appetite for reopening the placement of this species, or should we leave it at the _status quo ante_ until molecular data is available?

**Comment by crothfels** [comment]
I don't have any expertise to contribute here, but in the absence of decent evidence that it is a Lomariopsis, I would favor the "conservative" approach of leaving its generic placement unchanged (ie, continuing to recognize Thysanosoria). Lumping things, in the absence of strong evidence, is as disruptive as splitting them, in my opinion. So I vote we wait for molecular data...
